### PR TITLE
General UI improvements

### PR DESCRIPTION
**Pull request by looncraz**
Improved font awareness, enlarged logo, use of system colors (with live updating), and a few other minor modifications to make using Vision on modern high-res displays much more enjoyable.

I created a simple Jamfile as well, which just makes compilation several times faster.  The simple configure script is only required because it seems Haiku's jam has some bugs when trying to execute shell commands and my efforts to use the built-in Mkdir action were unfruitful.

There has been some incidental code cleanup, but nothing major.

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -1,2 +1,3 @@
 objects*/
 dist/Vision
+Vision
```
Comment (KapiX) [inline .gitignore]: I believe missing newline should be fixed.

File: src/AboutWindow.cpp [hunk 2]
```diff
@@ -97,6 +97,7 @@ AboutWindow::AboutWindow(void)
 				   "Bjorn Oksholen (GuinnessM)\n"
 				   "Jean-Baptiste M. Quéru (jbq)\n"
 				   "Humdinger\n"
+				   "looncraz"
```
Comment (KapiX) [inline src/AboutWindow.cpp]: Missing `\n`.

File: src/StatusView.cpp [hunk 3]
```diff
@@ -21,11 +21,10 @@
  *                 Todd Lair
  *                 Andrew Bazan
  *                 Jamie Wilkinson
+ *				   Joseph Groover <looncraz@looncraz.net>
```
Comment (KapiX) [inline src/StatusView.cpp]: Indentation.

**Comment by looncraz** [comment]
Thanks! Fixed.

**Comment by waddlesplash** [comment]
Very nice. Thanks!

**Pull request merged**